kansha/column/view.py

```diff
@@ -53,7 +53,7 @@ def render_column_overlay(self, h, comp, *args):
     with h.ul(class_='nav nav-list'):
         if not self.is_archive:
             with h.li:
-                onclick = "if (confirm(%(message)s)){%(callback)s;}" % {
+                onclick = "if (confirm(%(message)s)){YAHOO.kansha.app.hideOverlay();%(callback)s;}" % {
                     'message': ajax.py2js(_(u'The list will be deleted. Are you sure ?')),
```

Comment:
Does not work when french locale is activated. See master for a fix to your previous PR.